addons/runtime_debug_tools/scripts/debug_runtime_interaction_3d.gd

```diff
@@ -8,6 +8,9 @@ signal on_object_picked(node)
 var _selected_node : Node
 var _is_active = false
 var _previous_camera : Camera3D
+var _initialMouseMode : Input.MouseMode
+var _is_game_process_in_focus : bool
+var _set_initial_mouse_mode : bool
```

Comment:
(Same as 2D equivalent)

Request - rename this to `_restore_previous_mouse_mode`